components/text-input/TextInput.tsx

```diff
@@ -33,9 +37,13 @@ const TextInput: FC<Props> = ({
           type="text"
           value={value}
           onChange={(e) => {
+            if (customOnChange) {
```

Comment:
i dont think we need the chnages to this file in this pr

Reply:
I think it's ok.

It's a minor change that doesn't affect functionality. I can undo it if you really want, but it should allow an expanded use case that could easily be useful later.